SimTracker/TrackAssociation/plugins/TrackMCQuality.cc

```diff
@@ -39,7 +39,7 @@
 // class decleration
 //
 
-class TrackMCQuality : public edm::global::EDProducer<> {
```

Comment:
this class was never used.
It was supposed to enter in a different PR.
Still it is very useful to perform the analysis.
I plan to improve it later on adding Vertex information.